manifests/7/7S2P/Effie/CN/2.7.1/7S2P.Effie.CN.locale.zh-CN.yaml

```diff
@@ -1,43 +1,30 @@
-# Created with YamlCreate.ps1 v2.2.6 $debug=QUSU.CRLF.7-3-4.Win32NT
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.4.0.schema.json
-
-PackageIdentifier: 7S2P.Effie.CN
-PackageVersion: 2.7.1
-PackageLocale: zh-CN
-Publisher: 7S2P Inc.
-PublisherUrl: https://www.effie.co/
-# PublisherSupportUrl:
-PrivacyUrl: https://www.effie.co/privacy
-Author: 上海七加二科技有限公司
-PackageName: Effie
-PackageUrl: https://www.effie.co/
-License: 专有软件
-LicenseUrl: https://www.effie.co/terms
-Copyright: 版权所有 © 2023 上海七加二科技有限公司 保留所有权利
-# CopyrightUrl:
-ShortDescription: 把思想变成价值
-Description: Effie 是思想家专用的写作软件。无论是严肃写作，随手记录，亦或是把逻辑完善成思维导图，Effie 都是您明智的选择。
-# Moniker:
-Tags:
-- markdown
-- 写作
-- 大纲
-- 思维导图
-- 文档
-- 文稿
-- 文章
-- 笔记
-- 编辑器
-- 脑图
-# Agreements:
-ReleaseNotes: |-
-  - 改进的深色模式；
-  - 一些细节优化和 bug 修复。
-# ReleaseNotesUrl:
-# PurchaseUrl:
-# InstallationNotes:
-Documentations:
-- DocumentLabel: 常见问题
-  DocumentUrl: https://blog.effie.co/fq/
```

Comment:
I think these lines should not be deleted. 🤔 @vedantmgoyal2009 @russellbanks

Reply:
@russellbanks the lines from Default locale aren't deleted, so does komac treat default and other locales separately?

Reply:
Only in a really minor way. It checks for those values from the additional metadata JSON that you provide - I just hadn't done it so that it uses the previous/current value if you don't provide that. I'll push a fix